packages/sync-actions/src/products.js

```diff
@@ -124,6 +125,8 @@ function moveMasterVariantsIntoVariants(
   before: Object,
   now: Object
 ): Array<Object> {
+  /* eslint-disable no-param-reassign */
+  ;[before, now] = copyEmptyArrayProps(before, now)
```

Comment:
I'm not a fan of bending the linting rules here to mutate the passed args. Couldn't we just name them something else? Like 

`const [beforeCopy, nowCopy] = copyEmptyArrayProps(before, now)`

Reply:
@katmatt

Reply:
I saw the discussion above and agree that we should keep the passed argument names. But I think it would make sense to change them here instead.

Reply:
@MarkusAzer We use PRs to discuss code and to learn as a team together. We decided to use eslint to check our coding guidelines. And this means we should avoid bending the lint rules if we can. And it seems to be easy to fix it. That's why I suggest to fix it as @daern91 suggested it. What do you think about it?